test/functional/userAuth.test.js

```diff
@@ -38,6 +38,15 @@ describe('Authentication', () => {
           assert.equal(res.text, 'Found. Redirecting to /login')
         })
     })
+
+    it('should not redirect to login for static asset requests', () => {
```

Comment:
This seems like the right test, but would want to scope it per general comments about serving assets without auth as a default.